src/templates/index.hbs

```diff
@@ -1,37 +1,97 @@
+            <h2 class="intro">A convenient tool to share quotes with style</h2>
+        </div>
+      </div>
+  </header>
+  <section id="demo" style="margin-bottom:50px">
+    <div class="row text-center">
+          <p>To run the bookmarklet, drag the<a href="javascript:(function(){
+              if(document.getElementById('bookmarklet')) {
+                  document.getElementById('bookmarklet').remove();
+              } else {
+                  var importScript = (function (oHead) {
 
-<a href="javascript:(function(){
-    if(document.getElementById('bookmarklet')) {
-        document.getElementById('bookmarklet').remove();
-    } else {
-        var importScript = (function (oHead) {
+                    function loadError (oError) {
+                        throw new URIError('The script ' + oError.target.src + ' is not accessible.');
+                    }
 
-          function loadError (oError) {
-              throw new URIError('The script ' + oError.target.src + ' is not accessible.');
-          }
+                    return function (sSrc, fOnload) {
+                        var oScript = document.createElement('script');
+                        oScript.type = 'text\/javascript';
+                        oScript.onerror = loadError;
+                        if (fOnload) { oScript.onload = fOnload; }
+                        oHead.appendChild(oScript);
+                        oScript.src = sSrc;
+                    }
 
-          return function (sSrc, fOnload) {
-              var oScript = document.createElement('script');
-              oScript.type = 'text\/javascript';
-              oScript.onerror = loadError;
-              if (fOnload) { oScript.onload = fOnload; }
-              oHead.appendChild(oScript);
-              oScript.src = sSrc;
-          }
+                  })(document.head || document.getElementsByTagName('head')[0]);
 
-        })(document.head || document.getElementsByTagName('head')[0]);
+                  var style = document.createElement('link');
+                  style.rel = 'stylesheet';
+                  style.href = 'http://localhost:8080/css/base.css';
+                  document.head.appendChild(style);
 
-        var style = document.createElement('link');
-        style.rel = 'stylesheet';
-        style.href = 'http://localhost:8080/css/base.css';
-        document.head.appendChild(style);
+                  importScript('http://localhost:8080/js/bookmarklet.js');
 
-        importScript('http://localhost:8080/js/bookmarklet.js');
+                  bookmarklet.iFrame().init();
+              }
 
-        bookmarklet.iFrame().init();
-    }
-
-    })();"
-
-    onClick="javascript:(function(){console.log('hello!')})();">Add me</a>
+              })();"
 
+              onClick="javascript:(function(){console.log('hello!')})();"> Pullquote </a>link to your bookmarks bar.</p>
+      </div>
+    <div style="max-width:500px;display:block;margin:auto" >
```

Comment:
I get the desire to remove inline styles, especially this one, I'm hesitant, however, to keep up with the pattern of separate sass files for each individual concern. imho it's unnecessarily confusing for a project of this scale.